org.srb2.SRB2Kart.yaml

```diff
@@ -47,4 +49,16 @@ modules:
       - type: file
         path: org.srb2.SRB2Kart.desktop
       - type: file
-        path: org.srb2.SRB2Kart.metainfo.xml
\ No newline at end of file
+        path: org.srb2.SRB2Kart.metainfo.xml
+      - type: extra-data
+        url: https://github.com/STJr/Kart-Public/releases/download/v1.2/srb2kart-v12-Installer.exe
+        sha256: 57284de43a2889c0a60ce8fee77df22095fcd31beef6130cbeed1ead0bebbfec
+        filename: srb2kart-v12-Installer.exe
+        size: 200732615
+        commands:
+          - mkdir tmp
+          - cd tmp; 7za x ../srb2kart-v12-Installer.exe
+          - mkdir -p $FLATPAK_TODO_USER_VAR/sth/sth
+          - cp tmp/srb2.srb $FLATPAK_TODO_USER_VAR/sth/sth
+          - cp tmp/{patch,gfx,textures,chars,maps}.kart $FLATPAK_TODO_USER_VAR/sth/sth
+          - cp tmp/{music,sounds,bonuschars}.kart $FLATPAK_TODO_USER_VAR/sth/sth
```

Comment:
@ghisvail Any idea how to actually unpack it?  Do I need to abuse the first-run like your zenity bit does right now?

That seems unpleasant...  I want these commands (something like them) to happen at install time...

Reply:
> I want these commands (something like them) to happen at install time...

Which is what `extra-data` actually provides, together with a script called `apply_extra`. Look for its usage in the flatpak org. I believe repack of Slack or Spotify use it.

Reply:
Ah-ha!  This feature could really use better docs.